test/e2e/defaultbackend/default_backend.go

```diff
@@ -98,4 +98,35 @@ var _ = framework.IngressNginxDescribe("Default backend", func() {
 			Expect(resp.StatusCode).Should(Equal(test.Status))
 		}
 	})
+	It("should logging access to default backend when it enabled", func() {
+		f.UpdateNginxConfigMapData("enable-access-log-for-default-backend", "true")
+		host := "foo"
+		resp, _, errs := gorequest.New().
+			Get(f.IngressController.HTTPURL+"/somethingOne").
+			Set("Host", host).
+			End()
+
+		Expect(len(errs)).Should(Equal(0))
+		Expect(resp.StatusCode).Should(Equal(http.StatusNotFound))
+
+		logs, err := f.NginxLogs()
+		Expect(err).ToNot(HaveOccurred())
+		Expect(logs).To(ContainSubstring(`"/somethingOne"`))
+	})
+
+	It("should not logging access to default backend when it disabled", func() {
+		f.UpdateNginxConfigMapData("enable-access-log-for-default-backend", "true")
+		host := "bar"
+		resp, _, errs := gorequest.New().
+			Get(f.IngressController.HTTPURL+"/somethingTwo").
```

Comment:
You need to rebase with master and use the new API `f.GetURL(framework.HTTP)` here instead of `f.IngressController.HTTPURL`

Reply:
Thanks, but I don't know because I get it error in previous time:
test/e2e/defaultbackend/default_backend.go:105:9: f.IngressController undefined (type *framework.Framework has no field or method IngressController)